plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConfig.java

```diff
@@ -42,18 +42,24 @@
     public void testDefaults()
     {
         assertRecordedDefaults(recordDefaults(ClickHouseConfig.class)
-                .setMapStringAsVarchar(false));
+                .setMapStringAsVarchar(false)
+                .setJdbcTypesMappedToVarchar("")
+                .setClusterName(""));
     }
 
     @Test
     public void testExplicitPropertyMappings()
     {
         Map<String, String> properties = ImmutableMap.<String, String>builder()
                 .put("clickhouse.map-string-as-varchar", "true")
+                .put("jdbc-types-mapped-to-varchar", "REAL")
+                .put("cluster-name", "A6ch")
```

Comment:
nit: `A6ch` sounds the random string. I would recommend changing to `example-cluster` or something.